pulp_docker/app/urls.py

```diff
@@ -1,9 +1,19 @@
 from django.conf.urls import url
 
 from pulp_docker.app.views import (
-    VersionView
+    VersionView,
+    BlobManifestView,
+    TagView,
+    TagsListView
 )
 
 urlpatterns = [
     url(r'^v2/$', VersionView.as_view()),
+    url(r'^v2/(?P<path>.+)/tags/list$', TagsListView.as_view()),
+    url(r'^v2/(?P<path>.+)/manifests/(?P<digest>sha256:[A-Fa-f0-9]+)$',
```

Comment:
I think we can use `?i` to simplify the regex a little. `?iP<digest>sha256:[a-f0-9]+)$`

Reply:
can you point me to some docs on that? i am not familiar

Reply:
https://docs.python.org/3.7/library/re.html#re.IGNORECASE

Reply:
I am not sure how to make that work here. Perhaps you could make a followup PR to change it?

Reply:
I don't think its worth the effort if it didnt just work.